server/__tests__/app.test.js

```diff
@@ -114,3 +113,58 @@ afterAll(() => {
         })
     })
   })
+
+  describe('5. GET /api/reviews/:review_id/comments', () =>{
+
+    test("status:200, returns an array of comments matching the parametric review_id", ()=>{
+        return request(app)
+        .get('/api/reviews/2/comments')
+        .expect(200)
+        .then((response)=>{
+            const comments = response.body.comments;
+            expect(true).toBe(Array.isArray(comments));
```

Comment:
we don't need to explicitly check for an array here, as we are using forEach later it is implied that it must be an array